### DISCUSSION TITLE
Build.py script fails:  permissions error?

### DISCUSSION THREAD

**Discussion by bpickrel** [triton-inference-server/server#6554] [Q&A]
Can anyone help with a build error?  I'm trying out Triton-server for the first time and tried to build it with the script:

`./build.py -v --enable-all`

Here's the error I got.  Do I need an account or permissions that I don't know about?  Am I running the right script?

`ERROR: failed to solve: failed to copy: httpReadSeeker: failed open: failed to authorize: rpc error: code = Unknown desc = failed to fetch anonymous token: unexpected status from GET request to https://authn.nvidia.com/token?scope=repository%3Anvidia%2Ftritonserver%3Apull%2Cpush&service=registry: 401 Unauthorized`

This script had to run for several hours before it failed, so it's not easy to just try running it again.

**Comment by oandreeva-nv** [comment]
Could you please describe the environment in which you are trying to build the container?

**Reply by bpickrel** [reply]
This is in Ubuntu 20.04.6 LTS (Focal Fossa) on a desktop computer.
